server/startUp.js

```diff
@@ -23,7 +23,8 @@ async function getCSVs(dir) {
   const dirents = await readdir(dir, { withFileTypes: true });
   const files = await Promise.all(dirents.map((dirent) => {
     const res = path.resolve(dir, dirent.name);
-    return dirent.isDirectory() ? getCSVs(res) : res;
+    //return dirent.isDirectory() ? getCSVs(res) : res;
+    return dirent.isDirectory() ? "ignore" : res; //dont look for csv files recursively - look only for the files that RAMPART created in annotations pipeline
```

Comment:
"ignore" is not particularly nice. Maybe return null instead?

Also, do we want to keep commented out getCSVs line?